src/frontend/src/instance/log_handler.rs

```diff
@@ -84,6 +85,17 @@ impl PipelineHandler for Instance {
             .await
             .context(PipelineSnafu)
     }
+
+    async fn get_table(
+        &self,
+        catalog: &str,
+        schema: &str,
+        table: &str,
+    ) -> std::result::Result<Option<Arc<Table>>, catalog::error::Error> {
```

Comment:
nitpick: I would prefer passing `&QueryContext` instead of `catalog` and `schema` alone